include/seqan3/range/container/dynamic_bitset.hpp

```diff
@@ -0,0 +1,1897 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Enrico Seiler <enrico.seiler@fu-berlin.de>
+ * \brief A constexpr bitset implementation with dynamic size at compile time.
+ */
+
+#pragma once
+
+#include <seqan3/core/bit_manipulation.hpp>
+#include <seqan3/core/concept/cereal.hpp>
+#include <seqan3/core/detail/debug_stream_type.hpp>
+#include <seqan3/range/view/interleave.hpp>
+#include <seqan3/range/view/repeat_n.hpp>
+
+namespace seqan3
+{
+
+/*!\brief A constexpr bitset implementation with dynamic size at compile time.
+ * \implements seqan3::ReservableContainer
+ * \implements seqan3::Cerealisable
+ * \ingroup container
+ * \tparam capacity_   The capacity of the dynamic bitset
```

Comment:
```suggestion
 * \tparam capacity_ The capacity of the dynamic bitset
```

Reply:
Please do not name it `capacity_`. Call it `bitset_capacity`, maybe?

Reply:
Mhm `small_string` and `small_vector` also use `capacity_`.
I changed it to `bit_capacity`.